source/core.py

```diff
@@ -427,8 +427,9 @@ def main():
 	Finally, it starts the wx main loop.
 	"""
 	log.debug("Core starting")
-
-	ctypes.windll.user32.SetProcessDPIAware()
+	if globalVars.runningAsSource:

```

Comment:
Please add a comment here to highlight that this is handled via the manifest for packaged versions.